example/src/domain/App.js

```diff
@@ -1,9 +1,8 @@
-import { useElementSize }  from '@kaliber/use-element-size'
+import { useElementSize } from '@kaliber/use-element-size'
 import styles from './App.css'
 
 export default function App() {
-  const elementRef = React.useRef(null)
-  const { width, height } = useElementSize(elementRef)
+  const { size: { width, height }, ref: elementRef } = useElementSize()
```

Comment:
Wel jammer dat we zo de mogelijkheid verliezen om eenzelfde ref in meerdere hooks te hergebruiken, maar daar kunnen we als consequentie van de functie ref denk ik niet omheen werken. CC: @EECOLOR

Reply:
Als het daadwerkelijk nodig is zouden we een `useCompoundRef/useMultiRef` of iets dergelijks kunnen maken. Uit mijn hoofd is dat zoiets:

```js
function useMultiRef(...refs) {
  return React.useCallback(
    node => {
      refs.forEach(x => {
        if (typeof x === 'function') x(node)
        else x.current = node
      })
    },
    refs
  )
}
```